src/transformers/typescript.ts

```diff
@@ -152,9 +152,7 @@ function injectVarsToCode({
   // TODO investigate if it's possible to achieve this with a
   // TS transformer (previous attemps have failed)
   const codestores = Array.from(
-    contentForCodestores.match(
-      /\$[^\s();:,[\]{}.?!+\-=*/\\~|&%<>^`"'°§#1-9]+/g,
-    ) || [],
+    contentForCodestores.match(/\$[a-zA-Z_][\w_$]*/g) || [],
```

Comment:
This just makes more sense...
Matching for correct variables names instead of matching for things that are not variables

Reply:
Problem is that something like `$helloΩ` is a valid variable name but not matched by this regex.